src/lib/contracts.js

```diff
@@ -51,6 +51,9 @@ export {
 export {
   default as EthWrapperForBucketLender
 }                                       from '../../build/contracts/EthWrapperForBucketLender.json';
+export {
+  default as EthUnwrapperForBucketLender
```

Comment:
Nice, thanks for adding these.

Also, whenever we add new stuff to the js that we want to be used externally, we should increment the version number on `package.json` and `package-lock.json` (can do the package-lock one by npm installing after incrementing the package.json one)

This will cause a new version of the npm package to be published after the PR is merged to master, and will allow us to start using the changes in other repos